scripts/netContain/Dockerfile

```diff
@@ -17,30 +17,18 @@
 
 FROM ubuntu:16.04
 
-MAINTAINER Rajesh Nataraja <rajenata@cisco.com>
-
 # Make sure to Modify the Proxy Server values if required 
 # ENV export http_proxy=http://proxy.localhost.com:8080
 # ENV export https_proxy=https://proxy.localhost.com:8080
 
+RUN apt-get update \
```

Comment:
This produces a single layer smaller layer and the Dockerfile is cleaner. The apt lists clutter up the resulting image for no good reason without any benefit.

The official Docker Dockerfile from the docker/docker takes a similar approach to package installation and command formatting. The Docker docs for Dockerfile best practices recommend the same thing.